llvm/lib/Transforms/Utils/LoopSimplify.cpp

```diff
@@ -756,13 +751,8 @@ namespace {
       AU.addRequired<LoopInfoWrapperPass>();
```

Comment:
is the legacy pass still used in tree?

Reply:
Yes, unfortunately, as noticed by @nikic and the CI failure in test/CodeGen.